vignettes/usMaps.Rmd

```diff
@@ -74,6 +76,8 @@ shift_sp <- function(sp, scale, shift,
 
 ```
 
+For the United States, this function will get call the `shift_sp` function on the state/territory boundaries.
```

Comment:
Don't need to deal with it now, but at some point it might be good to switch this to `sf`.  Could be a good exercise for someone looking to learn `sf`?

Reply:
Yeah, I wanted to get this out because it was so helpful for the gage outage map. But, if someone looked at it and said, I could re-do that in `sf`, that would be great. In fact, someone at an unConf has already started I think